README.md

```diff
@@ -54,6 +54,7 @@ with the local datetime (e.g., `example-rule_2020-04-08_10:47:36.jsonl`).
 You can build this into a container with:
 
 ```
+sbt "project webapp" "dist"
```

Comment:
Thanks so much for all of this! Exciting!
Out of curiosity, is there any reason for having this command outside of the dockerfile?

Reply:
@BeckySharp sorry it took so long, I finally got around to taking a look. The reason it's separate is because you first need to create the application executable with sbt so that it can be added to the Docker image. Dockerfiles normally do not run build scripts... they are just for copying resources and running configurations for the actual image itself. The two builds (sbt and Docker) are conceptually different.

You wouldn't want to build the image inside of your Docker container because it's going to add a whole bunch of build artifacts to the image which can bloat the size and cause you to have files in your image that you did not intend to (it's minor here, but this can be a security concern).

We generally get around this problem by wrapping the two builds with `make`, however, that introduces more complexity/tools/dependencies on your contributors, so it's up to you if you'd like to do that. If you need some examples of how we do this at Two Six I can send you some repos.